generators/app/templates/infra/gcp/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "service_url" {
```

Comment:
Why have you put these outputs? Is it to have the url printed to the dev at the end of the `terraform apply` command? Do you think we should add similar outputs to the aws terraform?

Reply:
Yes we should, I create a ticket in backlog